specification/sovereign/Sovereign.Management/landingZoneRegistrationResourceProperties.tsp

```diff
@@ -0,0 +1,31 @@
+﻿using Azure.ResourceManager;
+using TypeSpec.Versioning;
+using Azure.Core;
+
+namespace Microsoft.Sovereign;
+
+@added(Versions.Preview20250227)
+@doc("The properties of landing zone registration resource type.")
+model LandingZoneRegistrationResourceProperties {
+  @visibility("read")
+  @doc("The status of a user-initiated, control-plane operation on the landing Zone registration resource type.")
```

Comment:
The state that reflects the current stage in the creation, updating, or deletion process of the landing zone registration resource type.